command/server.go

```diff
@@ -2589,6 +2589,7 @@ func setSeal(c *ServerCommand, config *server.Config, infoKeys []string, info ma
 
 		var wrapperInfoKeys []string
 		wrapperInfoMap := map[string]string{}
+		configSeal.Config = make(map[string]string)
```

Comment:
This doesn't look right. If `configSeal.Config` already had a value this will remove it.

The fix should address the real problem. Why did all KMS configs have a Config map before, but they do not do so now?

It seems to me that the fix should be in the `configureWrapper` function in kms.go, since it is related to the changes that @rculpepper did on PR #21116.